app/controllers/api/v1/items_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Api::V1::ItemsController < ApplicationController
+  # refactor idea: add set_merchant and set_item private methods, add a before_action for each to reduce redundancy of `merchant = Merchant.find(params[:merchant_id])` and `item = Item.find(params[:id])` - this can likely be applied to most controllers, so maybe even move it to the application controller
+  # refactor idea: remove nested if statements from index action
   def index
```

Comment:
I love both of these ideas (time permitted, of course). I wonder if there is a way to set those (item = Item.find(etc.)) easier since they are in every app we make. Seeing them so often makes me wonder if there isn't some Rails magic we just haven't been exposed to yet...